### PR TITLE
[Ready] [Could Use Sprites] add Antenna and Mind Reader powers, paranoia disability to genetics

### PR DESCRIPTION
**Pull request by PKPenguin321**
Antenna - It gives you an innate radio and an antenna sprite sticking out of your forehead.
Mind reader - You can read the minds of people on your screen. This will reveal semi-random bit of their past conversations, their intent, and their true name. You also get the antenna sprite. Can only be unlocked by combining Paranoia and Antenna.
Paranoia - You scream randomly and have a small chance to suffer from hallucinations now and then.

the power itself is done but i want to add a variant power or two that you can get through combining other stuff.

### IF YOU HAVE A BETTER SPRITE FOR THE ANTENNA PLEASE SEND IT MY WAY BECAUSE THE SPRITE REALLY SUCKS
MIND READER ALSO NEEDS AN ACTION BUTTON SPRITE
ALSO, i'd like the antenna to render above hair, so i need to remember to do that. if you know how to do that let me know

:cl: PKPenguin321
add: Added the Antenna power to genetics. It gives you a built-in radio.
add: Added the Mind Reader power to genetics. You can read the minds of people on your screen. This will reveal semi-random bit of their past conversations, their intent, and their true name. You also get the antenna sprite. Can only be unlocked by combining Paranoia and Antenna.
add: Added the Paranoia disability to genetics. You scream randomly, and have a small chance to suffer from hallucinations now and then.
/:cl:

(oh also added an admin-only dna injector for Radioactive)

### CODE REVIEW

**Comment by subject217** [comment]
how the fuck do chromosomes give someone a magic radio and an antenna head

**Comment by PKPenguin321** [comment]
how the fuck do you have an extra chromosome

**Comment by OverlyGeneric** [comment]
> how the fuck do chromosomes give someone a magic radio and an antenna head.

I think its more of insect antennae than a metallic rod. Some insects can detect frequencies from 50,000 to 100,000 Hertz.

If a human grew it, I would imagine it would most likely be able to "hear" frequencies ten times as much.

**Comment by PKPenguin321** [comment]
latest commit adds paranoia and mind reader

**Comment by YakumoChen** [comment]
Can mind reader be used to antag hunt?

**Comment by PKPenguin321** [comment]
All it tells you is their intent, their real name (if they're wearing a disguise or are covering their face and appear as Unknown you can see through it basically), and some random scattered past things they've said (up to 3 messages at a time, pulled from everything they've said in the round but GREATLY preferring more recent messages). Hypothetically if an antag was disguising themselves you could hunt for them with it, or if they had at some point said out loud "I am a traitor" and then didnt speak again for ages you might see that.

**Comment by YakumoChen** [comment]
Oh OK, the intent part confused me a bit. I was worried it let you see their notes.

**Comment by PKPenguin321** [comment]
Oh haha, no it's literally just "They intend to help someone..." or "They intend to disarm someone..."

**Comment by MMMiracles** [comment]
add a check that makes it so you can't read the thoughts of someone wearing a foil hat

**Comment by PKPenguin321** [comment]
Good idea, will have to add that soon

**Comment by anconfuzedrock** [comment]
hurr durr moths already have antenna

**Comment by OverlyGeneric** [comment]
> hurr durr moths already have antenna

double antennae 🤔

**Comment by PKPenguin321** [comment]
I'll look into changing the message. I think double antenna for moths is fine, but if it's a real issue it's possible to exclude the mutation from them. I'd rather not do that though

**Comment by Time-Green** [comment]
It’d be cool if you added Meta-Neural Enhancement with all the mind mutations, wich is pretty much tin foil hat mutation

**Comment by PKPenguin321** [comment]
that would be cool. would probably need to add a trait for it

**Comment by PKPenguin321** [comment]
Foil hats now block mindread (if either you OR the target is wearing one). I added an action button sprite for the mindread power (it's the old brain sprite with a slightly transparent paper sprite over it). I tweaked the antenna east and west sprites (they're still really bad though) and made the layer more fitting.

File: code/datums/mutations/combined.dm [hunk 1]
```diff
@@ -17,6 +17,11 @@
 	required = "/datum/mutation/human/strong; /datum/mutation/human/radioactive"
 	result = HULK
 
-/datum/generecipe/hulk
+<<<<<<< HEAD
```
Comment (PKPenguin321) [inline code/datums/mutations/combined.dm]: frick!

**Comment by Shardj** [comment]
Side note, do laser eyes not have instability? I can't see it in the files, might be worth adding onto the side of this pr.

**Comment by Time-Green** [comment]
Laser eyes aren't obtainable so there's no reason for them to have instability

**Comment by Shardj** [comment]
Thanks for the clarification

**Comment by ExcessiveUseOfCobblestone** [comment]
just FYI i'm iffy on the mindreading spell literally copying your logs every cast so I haven't wanted to review this lol.

**Comment by PKPenguin321** [comment]
>i'm iffy on the mindreading spell literally copying your logs every cast

It's really not as bad as it sounds, only copies the say logs which for most people is a list of around 50 text entries. Not very intensive at all

**Comment by lordsaladin** [comment]
are we my hero academia now?

**Comment by ExcessiveUseOfCobblestone** [comment]
this should be fine post-conflicts

**Comment by PKPenguin321** [comment]
hadn't realized it was conflicting, will fix

**Comment by ExcessiveUseOfCobblestone** [comment]
prob my fault

**Comment by PKPenguin321** [comment]
too bad we never got a pretty sprite, oh well

**Comment by Time-Green** [comment]
It’s ok man, have you looked closely at the genetic sequencer bar sprites? Legit nobody cares that I drew it in mspaint in 10 seconds and hit it with the bucket tool for the different states

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Comment by PKPenguin321** [comment]
image
mind read power is pretty great

**Pull request merged**